examples/common/python/connectors/direct/tcs_listener/tcs_work_order_handler.py

```diff
@@ -84,23 +96,17 @@ def process_work_order(self, input_json_str):
             - input_json_str is a work order request json as per TCF API 6.1 Work Order Direct Mode Invocation
         """
 
-        input_json = json.loads(input_json_str)
-        logger.info("Received Work Order request : %s", input_json['method'])
+        req = json.loads(input_json_str)
+        logger.info("Received Work Order request : %s", req['method'])
+        # save the full json for __process_work_order_submission
+        req["params"]["raw"] = input_json_str
 
-        response = {}
-        response['jsonrpc'] = '2.0'
-        response['id'] = input_json['id']
-        wo_id = str(input_json['params']['workOrderId'])
-
-        if(input_json['method'] == "WorkOrderSubmit"):
-            return self.__process_work_order_submission(wo_id, input_json_str, response)
-        elif(input_json['method'] == "WorkOrderGetResult"):
-            return self.__process_work_order_get_result(wo_id,
-                                                        input_json['id'],
-                                                        response)
+        data = json.dumps(req).encode('utf-8')
+        response = JSONRPCResponseManager.handle(data, self.dispatcher)
```

Comment:
I think this code which creates JSONRPCResponseManager could be move to "examples/common/python/connectors/direct/tcs_listener/tcs_listener.py"

Reply:
If the dispatcher moved into the "examples/common/python/connectors/direct/tcs_listener/tcs_listener.py", all handlers would need changing to employ this dispatcher.

As explained previously, 

> Changing all handlers at once would make it a PR too big. So rebasing other handlers in tcs_listener module would be proposed one by one later.

I would like do it step by step to ease reviewing @manojgop

Reply:
I didn't mean changing all handlers.  Can the json rpc dispatcher can be used at a level above in tcs_listener.py instead of using it here.

Reply:
Done~
Check the updated PR, please~ @manojgop